docs/user/how-to/work-with-code-hosted-on-launchpad/create-and-maintain-personal-branch.rst

```diff
@@ -8,44 +8,44 @@ Create and maintain a personal branch in Launchpad
 Branches hosted by Launchpad are usually asssociated with a project.
 
 If you'd prefer to upload a branch to Launchpad without it showing up in
-any project's branch listings, you can use the ``+junk``
+any project's branch listings, you can use the ``+git``
 pseudo-project. This can be useful if your branch is:
 
 -  for a project not yet in Launchpad
 -  not ready to be publicly associated with a project.
 
-``+junk`` branches work in a similar way to normal branches: anyone
+``+git`` branches work in a similar way to normal branches: anyone
 can create their own branch and they show up in `your own branch
 list <https://code.launchpad.net/people/+me>`__.
 
 However, there are some differences:
 
--  you can't propose a +junk branch for merging
+-  you can't propose a +git branch for merging
 -  they don't show up in any project's branch listing
--  you don't earn karma from +junk branches.
+-  you don't earn karma from +git branches.
 
-If you want to collaborate on a +junk branch with someone else, you need
+If you want to collaborate on a +git branch with someone else, you need
 to make the branch be owned by a team that both you and the other person
 are members of. Better still, when you are ready to collaborate, you
 should :ref:`create a
 project <registering-your-project>` and move
-the ``+junk`` branch into the project.
+the ``+git`` branch into the project.
 
-Create a +junk branch
+Create a +git branch
 ---------------------
 
-Adding a ``+junk`` branch to Launchpad is the same as creating any other
+Adding a ``+git`` branch to Launchpad is the same as creating any other
 hosted branch. Instead of using the project name in the branch URL, you
-use +junk.
+use +git.
```

Comment:
I only have a vague memory of the `+junk` thing.. I cannot recall whether this is related to bazaar.

Either we need to to some investigation in the code base or maybe someone in the team knows.